lib/ssl/src/ssl.erl

```diff
@@ -1105,6 +1105,42 @@ append_cipher_suites(Filters, Suites) ->
     Deferred = filter_cipher_suites(Suites, Filters), 
     (Suites -- Deferred) ++  Deferred.
 
+%%--------------------------------------------------------------------
+-spec signature_algs(Description, Version) -> [signature_algs()] when
+      Description :: default | all | exclusive,
+      Version :: protocol_version().
+
+%% Description: Returns possible signature algorithms/schemes
+%% for TLS/DTLS version
+%%--------------------------------------------------------------------
+
```

Comment:
signature_algs(Desc, 'dtlsv1.2') ->
    signature_algs(Desc, 'tlsv1.2').

Reply:
This is correct! DTLS-1.2 corresponds to TLS-1.2 and hence have the same signature algorithms.  For payload ciphers this is happens to be not true, but it is for signature algs.

Reply:
You can simplify the code here with one match for dtls, so you don't have to have it in 3 places.

Reply:
Ah, now I see what you mean!

Reply:
fixed!